hessian_test.go

```diff
@@ -171,3 +194,30 @@ func TestRequest(t *testing.T) {
 	doTestRequest(t, PackageRequest, Zero, []interface{}{"a", 3, true, []*Case{{A: "a", B: 3}}})
 	doTestRequest(t, PackageRequest, Zero, []interface{}{map[string][]*Case{"key": {{A: "a", B: 3}}}})
 }
+
+func TestHessianCodec_ReadAttachments(t *testing.T) {
+	body := &Response{
+		RspObj:      &CaseB{A: "A", B: CaseA{A: "a", B: 1, C: Case{A: "c", B: 2}}},
+		Exception:   nil,
+		Attachments: map[string]string{DUBBO_VERSION_KEY: "2.6.4"},
+	}
+	resp, err := doTestHessianEncodeHeader(t, PackageResponse, Response_OK, body)
+	assert.NoError(t, err)
+
+	pojoRegistry = POJORegistry{}
```

Comment:
why clean the pojo registry？

Reply:
for `ReadAttachments`, the pojos should not registry.

Reply:
the pojo should be recovered at the end of the unit test

Reply:
@zonghaishang  the pojo should be recovered at the end of the unit test

Reply:
yes, I will fix it .